src/governance.ts

```diff
@@ -35,6 +35,31 @@ export async function deployPayload({ filePath, provider }: DeployPayload) {
 interface CreateProposal extends DefaultInterface {
   payloadAddress: string;
 }
+
+interface CreateCalldataProposal extends DefaultInterface {
+  calldata: string;
+}
+
+/**
+ *
+ * @param {*} param0
+ * @returns proposalId
+ */
+export async function createCalldataProposal({
+  calldata,
+  provider,
+}: CreateCalldataProposal) {
+  // Create the proposal
+  const governance = new Contract(GOV, GOV_ABI, provider.getSigner(AAVE_WHALE));
```

Comment:
Only the tx sending part is not common between both methods, no?

Reply:
hmm yes could be unified, will do